Romanova_hw_4_.sql

```diff
@@ -0,0 +1,52 @@
+-- 1 Подключитесь к своей базе данных созданной на уроке
+-- 2 Создайте таблицу, которая отражает погоду в Вашем городе за последние 5 дней и включает следующее столбцы
+-- Id - первичный ключ, заполняется автоматически
+-- Дата - не может быть пропуском
+-- Дневная температура - целое число, принимает значения от -30 до 30
+-- Ночная температура - целое число, принимает значения от -30 до 30
+-- Скорость ветра - подумайте какой тип данных и ограничения необходимы для этого столбца
+Use IRYNAR_use;
+
+CREATE TABLE IF NOT EXISTS weather_city (
+	ID INT PRIMARY KEY AUTO_INCREMENT,
+	Weath_date DATE NOT NULL,
+	Day_temp INT,
+	Night_temp INT,
+	Wind_speed DECIMAL(5,2),
+	CHECK (Day_temp BETWEEN -30 AND 50),
+	CHECK (Night_temp BETWEEN -30 AND 50)
```

Comment:
А почему до 50?
Двойные неравенства тоже будут работать.
```suggestion
	CHECK(-30 >= Day_temp >= 30),
	CHECK(-30 >= Night_temp >= 30)
```